modules/oe_theme_helper/src/TwigExtension/Filters.php

```diff
@@ -61,16 +65,22 @@ public function toLanguageName($language_code): string {
    *
    * @return string
    *   The native language name.
-   *
-   * @throws \InvalidArgumentException
-   *   Thrown when the passed in language code does not exist.
    */
-  public function toNativeLanguageName($language_code): string {
+  public function toNativeLanguageName($language_code) {
```

Comment:
Do you think it would b possible to add a type cast to the argument `$language_code` and to the function return as well ?

Reply:
The function can return a string or a TranslatableMarkup so I should pbly update the return doc to:
`@return string|\Drupal\Core\StringTranslation\TranslatableMarkup`
But I don't think I can set the return type since it's multiple.

For the argument do you mean something like this?:
`public function toNativeLanguageName(string $language_code) {`
I think that should be ok.

Reply:
Thanks Hervé.

I think it's always better to have functions that returns on single type of value, but sometimes it's not possible.

For the argument, yes, it's fine, thanks mate.

Reply:
I made the changes, let me know.